install.md

```diff
@@ -230,3 +230,113 @@ the [trouble shooting page](https://julialang.github.io/IJulia.jl/stable/manual/
 If you try to open an existing notebook (stored in a ".ipynb" file), it might refer to an
 older Julia version, resulting in a "Kernel error"; the solution is then to select
 a different kernel from the menu.
+
+## System-wide installation of Oscar
+
+It is possible to install Oscar and its cornerstones only once on your system,
+and to let several people use this installation.
+For that, the following `bash` scripts can be used.
+
+<div class="clickdesc">
+
+<details>
+<summary>
+Install Oscar system-wide.
+</summary>
+Enter the following commands into a file (for example <code>oscar_systemwide_install</code>),
+adjust the paths for the variables <code>julia_for_oscar</code> and <code>central_depot</code>,
+and then run the script in a terminal (with administrator rights).
+{% highlight bash %}
+#!/bin/bash
+
+# Specify the intended version of Julia.
+julia_for_oscar=/users/oscar/julia-1.6.1/bin/julia
+
+# Specify the intended location of the central Oscar installation.
+central_depot=/users/oscar/JULIA_DEPOT
+
+# Set the Julia variables that control the location of packages.
+# (Do not admit the current user's own depot path.)
+export JULIA_DEPOT_PATH=${central_depot}
+export JULIA_LOAD_PATH="@:@v#.#:@stdlib:${central_depot}/environments/globalenv"
+
+# Let Julia install and precompile the packages.
+${julia_for_oscar} -e 'using Pkg; \
+Pkg.add("Oscar"); using Oscar; \
+Pkg.add("GAP"); using GAP; \
+Pkg.add("Nemo"); using Nemo; \
+Pkg.add("Hecke"); using Hecke; \
+Pkg.add("Singular"); using Singular; \
+Pkg.add("Polymake"); using Polymake; \
+Pkg.add("AbstractAlgebra"); using AbstractAlgebra; \
```

Comment:
Why is it necessary to install all of these; doesn't just Oscar suffice?

Reply:
If only Oscar is added to the environment then the other packages are installed physically but they are not added to the `Project.toml`. A user calling `using GAP` (for example) will then get the message that first one has to call `Pkg.add("GAP")`, and trying this will result in the error message that the user has no write permission for the central installation.

Reply:
I think I don't understand how this depot is intended to be used. I thought it was mainly about caching all the large artifacts and package files, so users don't need to have them in their home directories (which might also be on NFS)

But it seem you are suggesting everyone should be using *that same environment*? I would have thought everyone is still expect to at least once `Pkg.add("Oscar")`  (or, of course, `Pkg.add("GAP")`). But OK, I guess that doesn't mesh well with "having Oscar up-to-date all the time". On the other hand, *that* doesn't mesh well with "please don't break my working Oscar behind my back"... Hrm

Reply:
If the idea is to share just the artifacts then we can provide a recommended `Overrides.toml` file that points to directories in the `JULIA_DEPOT/artifacts` in question.
In fact, the two approaches do not exclude each other.

Reply:
But Julia allows for a stack of active depots, see https://docs.julialang.org/en/v1/base/constants/#Base.DEPOT_PATH -- note that` /usr/local/share/julia` is on that default list. So if one just puts artifacts in there, they should be loadable by any Julia user on that system without any additional effort.

Reply:
@fingolfin I understand your idea as follows: First make an Oscar installation. Then take the artifacts that have been installed during this installation, and move them to the "architecture-specific shared system directory" that can be found in `DEPOT_PATH`. This way one need not provide an `Overrides.toml`. For updating an existing "shared Oscar installation", one repeats this process. Is this interpretation correct?

Reply:
I have made more experiments.

Providing the necessary artifacts inside the "architecture-specific shared system directory" as sketched above works.
(Well, some artifact seems to be downloaded later on, and hence ends up in the user's `.julia/artifacts` directory; I think this is not desirable.)

However, I have not found a way to make the idea to share artifacts compatible with the idea to have a complete installation of Oscar that is shared between users. (I tried to use the artifacts inside the "architecture-specific shared system directory" by the central installation, but some packages try to find the artifacts in the directory of the central installation.)